scripts/xy_grid.py

```diff
@@ -244,6 +244,18 @@ def cell(x, y):
 
             return process_images(pc)
 
+        if not x_opt.label == 'Nothing':
+            p.extra_generation_params["XY Plot X Type"] = x_opt.label
+            p.extra_generation_params["X Values"] = '{' + x_values + '}'
+            if x_opt.label in ["Seed","Var. seed"] and not no_fixed_seeds:
+                p.extra_generation_params["Fixed X Values"] = '{' + ", ".join([str(x) for x in xs])+ '}'

```

Comment:
Do you mean "no" to using the literal keys "X Values" and "Y Values" here? I don't have a strong opinion on that, I was just thinking that since the previous keys were "XY Plot X Type", the "XY Plot" could be taken for granted, but I can see how it might be better to explicitly say "XY Plot X Values", and "XY Plot Fixed X Values", etc. as "X Values" could easily collide with something else in the future when parsing infotext.